src/docfx/lib/HtmlUtility.cs

```diff
@@ -21,19 +21,16 @@ internal static class HtmlUtility
         private static readonly string[] s_allowedStyles = new[] { "text-align: right;", "text-align: left;", "text-align: center;" };
 
         // todo remove `class` & `hidden` & `tabindex` after we can know it is is generated by ourself
-        private static readonly HashSet<string> s_globalAllowedAttributes = new HashSet<string>(StringComparer.OrdinalIgnoreCase) { "id", "itemid", "itemprop", "itemref", "itemscope", "itemtype", "part", "slot", "spellcheck", "title", "role", "class", "hidden", "tabindex" };
```

Comment:
can we remove class & hidden & tabindex here since currently we can make sure the attributes is in user HTML?